### ISSUE TITLE
With Nvidia 304xx driver I can't login

### ISSUE THREAD

**Issue by cizordj** [budgie-desktop/budgie-desktop#1307]
**Operating System**:  Solus 3 Shannon
**Budgie version**: Budgie 10.4

Before submitting *ANY* issue, please run the following command to
verify that you have, at minimum, Budgie `10.2.7`, to provide the
exact version of your Budgie installation:

    budgie-desktop --version
budgie-desktop 10.4

**The issue encountered**

When I installed the Nvidia's driver (version 304xx) via DoFlicky the Budgie Desktop couldn't be initiated, the login screen works normally but when I type the password and wait for the interface to be loaded it doesn't show up and get back to login screen, my budgie desktop version is 10.4 but it happens with earlier versions too.
 I'm now using Solus OS with MATE and Openbox's interface and they're working very well with Nvidia driver installed.

**Steps to reproduce the issue**

First, to have a legacy video card like mine. Open 'Hardware Drivers' in menu, choose 'Nvidia GRAPHICS DRIVER (304.xx Series)' (this is the only compatible version) and install it. After rebooting the issue happens. 
There is more information about my machine here:
![hei](https://user-images.githubusercontent.com/32869222/35590287-82fa6fea-05ed-11e8-9be5-d39d0c82cce4.png)
(ASAP I'll improve my RAM and CPU capacity)

  I tried installing these drivers manually directly from nvidia website and the same thing happens.

I appreciate your job, thanks in advance =)

**Comment by ikeydoherty** [comment]
This isn't a Budgie issue. We do not support the installation of drivers from NVIDIA's website in Solus, only the ones in our repos. Please make sure you're using the correct (and supported) NVIDIA drivers, and only install the ones for your kernel. I'm a bit puzzled about your various Budgie versions (like, how do you have them on a rolling release..?) and I'm concerned that your installation of NVIDIA manually will bork your system beyond any package manager repair.

I'd advise a fresh install with all the latest updates (all of them), reboot, and **then** install graphics drivers. From repos.

**Comment by cizordj** [comment]
I'm sure about the correct driver because I used the same version on Windows 10 and others distros. I'm not running Solus with Nvidia's manual installation anymore.

> I'm a bit puzzled about your various Budgie versions (like, how do you have them on a rolling release..?) 
=p I don't really know, where did you see that? I upgraded the system many times on software center and sometimes with `# eopkg upgrade` maybe this caused it.

> I'd advise a fresh install with all the latest updates (all of them), reboot, and then install graphics drivers. From repos.
 I'll do that, what I used to do is: to install a fresh copy of Solus Budgie, install the drivers with DoFlicky, upgrade the system and then reboot it. Everything at once. 
Hope that works, thanks
